### ISSUE TITLE
CC a11y: publish Breadcrumb checklist - draft content

### ISSUE THREAD

**Issue by finekatie** [uswds/uswds-site#2824]
### A11Y
- [x] Review checklist content
- [x] Final approval of checklist content

### Content
- [x] Draft checklist content and send for review with A11Y
- [x] Final edits to checklist content

### Dev
- [ ] Create PR for checklist content edits and request review by A11Y and CS

## References
[Critical Checklist spreadsheet](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)
[List of all components](https://docs.google.com/spreadsheets/d/1ASzZ80oI6WVWAifqPO6gYAqrsRaiZs_qGTOStuQPyKs/edit?usp=sharing)

**Comment by finekatie** [comment]
Hi @amycole501  and @alex-hull , are you ready for me to start gherkins on this?

**Comment by amycole501** [comment]
Yes they should be ready for you, Katie

**Comment by finekatie** [comment]
80% done. Should be able to finish Monday.

**Comment by finekatie** [comment]
Breadcrumbs are done and ready for review, @amycole501  and @alex-hull :)

**Comment by alex-hull** [comment]
Reviewed these gherkins and left a few suggestions in the spreadsheet! @finekatie @amycole501

**Comment by amycole501** [comment]
I also reviewed and added a couple of follow up notes. Thanks!

**Comment by finekatie** [comment]
@amycole501 and @alex-hull , I reviewed and accepted/resolved comments. Are we ready for dev review before PR?

**Comment by alex-hull** [comment]
Yes, if everything has been addressed, then we can pass it on!

**Comment by finekatie** [comment]
Hey @brunerae , what Dev gets this assigned for creating PR now?

**Comment by alex-hull** [comment]
@amyleadem @cathybaptista @RachelCorsino These gherkins are ready for y'all to create the dev issue!

**Comment by brunerae** [comment]
This is live https://designsystem.digital.gov/components/breadcrumb/accessibility-tests/
